lib/app/console_app.rb

```diff
@@ -46,17 +54,20 @@ def register_connector
       if id.present?
         puts "You already have registered a connector with ID: #{id}. Registering a new connector will not use the existing one."
         puts 'Are you sure you want to continue? (y/n)'
-        return unless gets.chomp.casecmp('y').zero?
+        return false unless gets.chomp.casecmp('y')&.zero?
```

Comment:
Why `return false` here?

If it's needed, probably other return statements in the method should also return a value

Reply:
Good catch. Yeah they should. I missed one case.